_posts/2018-09-10-tutorial.md

```diff
@@ -93,18 +105,6 @@ This will build the ``master`` and ``worker`` docker images and push them to the
 
 ### Installing MLBench
 
-To install MLBench, checkout the current repository:
-
-```shell
-$ git clone git@github.com:mlbench/mlbench.git
-```
-
-Enter the newly created directory
-
-```shell
-$ cd mlbench
-```
-
 Now create a file called ``myvalues.yaml`` in this directory with the following content:
```

Comment:
It's not yet part of the source, instead I added a reference to the charts/mlbench/values.yaml that can be used as a template (since that's the intended use of the file in helm charts)